modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/RichTextField.testcase

```diff
@@ -130,7 +161,28 @@ definition {
 	test SetRichTextSearchableAsDisabled {
 		property portal.acceptance = "false";
 
-		// TODO LPS-98076 SetRichTextSearchableAsDisabled pending implementation
+		DataEngine.addField(
+			fieldFieldLabel = "Test Searchable as Disabled",
+			fieldName = "Rich Text");
+
+		DataEngine.editFieldSearchable(
+			fieldFieldLabel = "Test Searchable as Disabled",
+			searchableType = "Disable");
+
+		WebContentStructures.saveCP();
+
+		WebContentNavigator.gotoEditStructure(structureName = "New Structure");
+
+		AssertClick.assertPartialTextClickAt(
+			key_fieldFieldLabel = "Test Searchable as Disabled",
+			locator1 = "DDMEditStructure#FORM_FIELD_CONTAINER",
+			value1 = "Test Searchable as Disabled");
+
+		Navigator.gotoNavTab(navTab = "Advanced");
+
+		FormFields.viewRadioValue(
+			fieldName = "indexType",
+			fieldValue = "Disable");
```

Comment:
hey @nicolas-sss , this test was already sent at https://github.com/alexcandidos/liferay-portal/pull/254/files